src/openforms/appointments/contrib/jcc/tests/test_plugin.py

```diff
@@ -0,0 +1,144 @@
+import os
+from datetime import date, datetime
+
+from django.test import TestCase
+
+import requests_mock
+from zeep.client import Client
+
+from openforms.appointments.contrib.base import (
```

Comment:
Please use relative imports within the same app

```suggestion
from ...base import (
```

Reply:
Oh, weird I usually do that. Probably an auto import.